server/src/routes/api/v1/parksRouter.js

```diff
@@ -1,41 +1,55 @@
-import express from "express"
-import Park from "../../../models/Park.js"
-import cleanUserInput from "../../../services/cleanUserInput.js"
-import objection from "objection"
-const { ValidationError } = objection
+import express from "express";
+import objection from "objection";
+const { ValidationError } = objection;
+
+import cleanUserInput from "../../../services/cleanUserInput.js";
+import Park from "../../../models/Park.js";
+import parkReviewsRouter from "./parkReviewsRouter.js";
+import ParkSerializer from "../../serializer/ParkSerializer.js";
 
 const parksRouter = new express.Router();
 
 parksRouter.get("/", async (req, res) => {
   try {
-    const parks = await Park.query()
-    return res.status(200).json({ parks: parks })
-  } catch(error) {
-    return res.status(500).json({ errors: error })
+    const parks = await Park.query();
+    const serializedParks = [];
+
+    for (const park of parks) {
+      const serializedPark = await ParkSerializer.showData(park);
+      serializedParks.push(serializedPark);
+    }
+    return res.status(200).json({ parks: serializedParks });
+  } catch (error) {
+    return res.status(500).json({ errors: error });
   }
-})
+});
 
-parksRouter.get("/:id", async (req,res) => {
-const parkId = req.params.id;
+parksRouter.get("/:id", async (req, res) => {
+  const parkId = req.params.id;
   try {
     const park = await Park.query().findById(parkId);
-    return res.status(200).json({ park: park  })
+    const serializedPark = await ParkSerializer.showData(park);
+    return res.status(200).json({ park: serializedPark });
   } catch (error) {
-    return res.status(500).json({ errors: error})
+    return res.status(500).json({ errors: error });
   }
-})
+});
 
 parksRouter.post("/", async (req, res) => {
-  const { body } = req
-  const formInput = cleanUserInput(body)
+  const { body } = req;
+  const formInput = cleanUserInput(body);
+
   try {
-    const newPark = await Park.query().insertAndFetch(formInput)
-    return res.status(201).json({park: newPark})
-  } catch(error) {
+    const newPark = await Park.query().insertAndFetch(formInput);
```

Comment:
This new park should be passed through the serializer to remove things like the timestamps